src/application/App.tsx

```diff
@@ -144,6 +144,12 @@ export const App = () => {
     setSelectedClientId(clientIds[0]);
   }
 
+  useEffect(() => {
+    if (clients.length) {
+      send({ type: "connect" });
+    }
+  }, [send, clients]);
```

Comment:
```suggestion
  }, [send, clients.length]);
```

No need to rely on referential equality here when all we rely on is the length, right?

Reply:
Good call!